### DISCUSSION TITLE
A solution to hide input box in the pop-up window

### DISCUSSION THREAD

**Discussion by reynaldmorel** [nhn/tui.calendar#983] [General]
Hello,

I found a solution by modifying the 'tui-calendar.css' file.

For example, if you want hidden input box << Location, >>, modify as follows :
.tui-full-calendar-popup-section-item.tui-full-calendar-section-location {
OLD -> /display: block;/
display: none;
}

Reynald

**Comment by adhrinae** [comment]
Thanks for the tip. I moved your issue to the discussion.

**Reply by reynaldmorel** [reply]
with pleasure
